<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPasswordDialog</class>
 <widget class="QDialog" name="ExportPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Passwords</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="selectAllCheckBox">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <widget class="QGroupBox" name="fileTypeGroupBox">
       <property name="title">
        <string>File type</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="csvCheckBox">
          <property name="text">
           <string>CSV</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="csvFilenameLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>CSV filename</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="jsonCheckBox">
          <property name="text">
           <string>JSON</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="jsonFilenameLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>JSON filename</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="xmlCheckBox">
          <property name="text">
           <string>XML</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="xmlFilenameLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>XML filename</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
